src/oc-id/spec/controllers/zendesks_controller_spec.rb

```diff
@@ -29,17 +29,43 @@
       end
 
       context 'when signed in' do
+        let(:testuser) do
+          User.new({
+            :username => 'jimmy',
+            :email => 'jim.kirk@federation-captains.org',
+          })
+        end
         before :each do
           allow(controller).to receive(:signed_in?).and_return(true)
-          allow(controller).to receive(:current_user).and_return('testuser')
+          allow(controller).to receive(:current_user).and_return(testuser.email)
+          allow(controller).to receive(:current_user).and_return(testuser)
```

Comment:
This seems weird - we're allowing it to receive :current_user twice; the first return is email and thee second is user object. Is that intentional?

Reply:
fixed! thanks!